src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to use a wallet
+description: A guide explaining how to send, receive tokens and connect to web3 projects.
+lang: en
+---
+
+# How to: Use a wallet
+
+Learn how to operate all the basic functions of a wallet. If you don’t have one yet, check out our [How to: "Register" an Ethereum account](/guides/how-to-register-an-ethereum-account/).
+
+## Open your wallet
+
+You should see a dashboard that will likely show your balance and contain buttons to send and receive tokens. 
+
+## Receive cryptocurrency
+
+Do you want to receive crypto into your wallet?
+
+Each Ethereum account has its own receiving address which is a unique sequence of numbers and letters. The address functions like a bank account number. Ethereum addresses will always start with “0x”.
+
+You need to provide the sender with your receiving address. Many wallet apps let you copy to clipboard the address or show a QR code to scan for easier usage. Avoid typing any Ethereum address manually. This is unsafe, and can easily lead to clerical errors and lost funds.
+
+1. Open your wallet app.
+2. Click on "Receive" or similarly worded option on the dashboard
+3. Copy your Ethereum address to clipboard.
+4. Provide the sender with your receiving Ethereum address.
+
+This is a general pattern, and different apps may vary or use different language. It should also be similar process if you are trying to send your funds from an exchange. 
```

Comment:
Propose to move this above the bullet points prior as the transition from "avoid manual typing" to the steps is rather sudden. Suggested format below:

The steps listed below is a general pattern for receiving cryptocurrency, and different apps may vary or use different language. It should also be _a_ similar process if you are trying to send your funds from an exchange.

1. Open your wallet app.
2. Click on "Receive" or similarly worded option on the dashboard
3. Copy your Ethereum address to clipboard.
4. Provide the sender with your receiving Ethereum address.